techtrix:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # Position Controllers ---------------------------------------
  base_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: base_joint
    pid: { p: 100.0, i: 0.01, d: 10.0 }

  lifting_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: lifting_joint
    pid: { p: 750.0, i: 0.01, d: 10.0 }

  rotating_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: rotating_joint
    pid: { p: 100.0, i: 0.01, d: 10.0 }
